include/WCSimGenerator_Radioactivity.hh

```diff
@@ -24,133 +31,82 @@ class WCSimGenerator_Radioactivity
 		void Configuration(G4int iScenario, G4double dLifeTime=0);
 		
 		G4ThreeVector GetRandomVertex(G4int tSymNumber);
-		G4double GetMeanActivity() { return fIntegral; }
-		G4double GetIDVolume() { return (fR_max*fR_max*TMath::Pi()*fZ_max*2.); }
+		G4double GetMeanActivity()		{ return fConcentrationID; }
+		G4double GetMeanActivityReco()	{ return fConcentrationReco; }
+		G4double GetIDVolume()			{ return (fR_max*fR_max*TMath::Pi()*fZ_max*2.); }
 		
 		
 	private:
 	
-		static G4double ZFit_R210(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double ZFit_R190(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double ZFit_R170(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double ZFit_R140(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double ZFit_R070(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double ZFit_R040(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double ZFit_R020(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double ZFit_R000(G4double x, G4double Lambda, G4double BinConversion);
-
-		static G4double R2Fit_ZpM(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double R2Fit_ZmM(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double R2Fit_Z16(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double R2Fit_Z13(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double R2Fit_Z10(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double R2Fit_Z00(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double R2Fit_Z14p(G4double x, G4double Lambda, G4double BinConversion);
-		static G4double R2Fit_Z16p(G4double x, G4double Lambda, G4double BinConversion);
+		// Diffusion functions
+		static double DiffusionZ(double x, double origin, double factor);
+		static double DiffusionR(double x, double origin, double factor);
+		
+		// Concentration functions
+		static double RadonFormulaR(double* val, double* par);
+		static double RadonFormulaZ(double* val, double* par);
 		
 		// Function
 		TF2* thRnFunction;
 		G4double fIntegral;
+		G4double fIntegralReco;
```

Comment:
What does `Reco` mean in this context?

Reply:
I modified the name of the "reco" variables, it was for the "reconstruction volume" I renamed them as "FV" (fiducial volume).